packages/tools/devtools/devtools-core/src/messaging/devtools-messages/ContainerList.ts

```diff
@@ -9,6 +9,8 @@ import type { IDevtoolsMessage } from "../Messages.js";
 /**
  * Encapsulates types and logic related to {@link ContainerList.Message}.
  *
+ * @remarks Handles both containers and data objects registered with the devtools.
+ *
```

Comment:
This comment still references "data objects" rather than "container runtimes". That said, I don't know that this note is necessary. This list is really about the registered container-level devtools instances (`IContainerDevtools`), which hasn't changed (it just has another implementation).